sos/report/plugins/openvswitch.py

```diff
@@ -32,33 +33,10 @@ def setup(self):
 
         dpdk_enabled = self.collect_cmd_output(
             "ovs-vsctl -t 5 get Open_vSwitch . other_config:dpdk-init")
-        check_dpdk = (dpdk_enabled["status"] == 0 and
-                      dpdk_enabled["output"].startswith('"true"'))
-        check_6wind = any([self.is_installed(p) for p in
-                           ['6windgate-fp', 'nuage-openvswitch']])
-        actl = "ovs-appctl"
-
-        files_6wind = [
-            "/etc/systemd/system/multi-user.target.wants/openvswitch.service",
-            "/etc/sysctl.d/60-6wind-system-auto-reboot.conf",
-            "/etc/openvswitch/system-id.conf",
-            "/etc/openvswitch/*.db",
-            "/etc/ld.so.conf.d/linux-fp-sync-fptun.conf",
-            "/etc/NetworkManager/conf.d/fpn0.conf",
-            "/etc/default/openvswitch",
-            "/etc/logrotate.d/openvswitch",
-            "/etc/linux-fp-sync.env",
-            "/etc/fp-daemons.env",
-            "/etc/fp-vdev.ini",
-            "/etc/fpm.env",
-            "/etc/6WINDGate/fp.config",
-            "/etc/6WINDGate/fpnsdk.config",
-            "/etc/dms.d/fp-dms.conf",
-            "/etc/dms.d/fpmd-dms.conf",
-            "/etc/dms.d/fpsd-dms.conf",
-            "/etc/fast-path.env",
-            "/etc/fps-fp.env",
-        ]
+        self.check_dpdk = (dpdk_enabled["status"] == 0 and
```

Comment:
This was a rather large function and had to refactor it a bit to solve too-many-locals, too-many-statements, etc.